packages/web-components/src/components/ic-breadcrumb-group/ic-breadcrumb-group.stories.mdx

```diff
@@ -453,3 +453,79 @@ import BreadcrumbReadme from "../ic-breadcrumb/readme.md";
     `}
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  appearance: "default",
+  backBreadcrumbOnly: false,
+  collapsed: false,
```

Comment:
The collapsed prop doesn't take immediate effect.
The styling for the collapsed section also doesn't take into account the current appearance. It always takes the default styling
https://github.com/mi6/ic-ui-kit/assets/113986285/0d7db269-aab0-4804-8bc4-8f37ad76190e

When collapsed is set as false after the steps taken in the video above, the breadcrumb group shows incorrectly. Based on this, I think collapsed prop
<img width="496" alt="Screenshot 2024-06-19 at 13 33 25" src="https://github.com/mi6/ic-ui-kit/assets/113986285/abdf2531-6f9c-4eaf-a0c4-b28ab76a0fe5">

Reply:
I have a ticket open for issues with the collapsed prop, so I'll add this to it